Gemfile

```diff
@@ -7,6 +7,10 @@ gemspec
 gem "handsoap", "~>0.2.5", :require => false, :git => "https://github.com/ManageIQ/handsoap.git", :tag => "v0.2.5-5"
 gem "rubywbem",            :require => false, :git => "https://github.com/ManageIQ/rubywbem.git", :branch => "rubywbem_0_1_0"
 
+# This is needed because some of the smart analysis tests scripts, insde the 'lib/gems/pending/MiqVm/test' directory,
+# use the oVirt provider inventory classes.
+gem "manageiq-providers-ovirt", :require => false, :git => "https://github.com/ManageIQ/manageiq-providers-ovirt.git", :branch => "master"
```

Comment:
Once this is merged, we'll need to open GH issue in manageiq-releasen repo to update the branch name when we cut gaprindashvili.